src/index.js

```diff
@@ -29,11 +29,8 @@ const getOutput = (changes, projects) => {
         core.error('Only pull requests events can trigger this action');
     }
 
-    const projects = core.getInput('projects', {required: true}).split(' ');
-
-    if (!Array.isArray(projects)) {
-        core.error('Projects list is not defined');
-    }
+    const domainsString = core.getInput('domains', {required: true});
+    const domains = JSON.parse(domainsString);
```

Comment:
What if `domains` are not parseable to JSON or you don't wanna handle it and just let it throw whatever comes?

Reply:
It will fail and stop the process with an unexpected JSON token error, I don't really wanna invest in the safety of this action much. Unless it starts failing.